Project.toml

```
@@ -1,22 +1,22 @@
 name = "CarlemanLinearization"
 uuid = "4803f6b2-022a-4c1b-a771-522a3413ec86"
-version = "0.2.1"
+version = "0.3.0"
 
 [deps]
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+MultivariatePolynomials = "102ac46a-7ee4-5c85-9060-abc95bfdeaa3"
 Requires = "ae029012-a4dd-5104-9daa-d747884805df"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 
 [compat]
-DynamicPolynomials = "0.3, 0.4"
+DynamicPolynomials = "0.4"
 MultivariatePolynomials = "0.3, 0.4"
```

```suggestion
DynamicPolynomials = "0.3, 0.4"
MultivariatePolynomials = "0.3, 0.4"
```
The change in `DynamicPolynomials` from v0.3 to v0.4 was the change in `MultivariatePolynomials` from v0.3 to v0.4. So it does not make sense to support `MultivariatePolynomials` v0.3 but not the other.
But see also the comment below.

---

:+1:  Then I'll just only keep `v0.4` in both packages.